frontend/src/editor/player.tsx

```diff
@@ -92,9 +92,8 @@ export function PlayerBar({ documentId, editor }: { documentId: string; editor:
   // skip to a timestamp if the user clicks on a word in the transcript. The corresponding event is
   // dispatched in transcription_editor.tsx
   useEvent<TextClickEvent>(TEXT_CLICK_EVENT, (e) => {
-    progressCallback();
-    if (e.detail.text.start) {
-      waveSurferRef.current?.seekTo(e.detail.text.start / waveSurferRef.current.getDuration());
+    if (e.detail.start != undefined) {
```

Comment:
good catch :D. but don't we still need to call progressCallback after that?

Reply:
`progressCallback` is already called after seeking due to listening to the seek events, so I am not sure what calling it here aswell would really do?